py/frequenz/client/weather/_types.py

```diff
@@ -127,7 +128,7 @@ def to_ndarray_vlf(
         validity_times: list[dt.timedelta] | None = None,
         locations: list[Location] | None = None,
         features: list[ForecastFeature] | None = None,
-    ) -> np.ndarray:
+    ) -> np.ndarray[tuple[typing.Literal[3], typing.Any], np.dtype[np.float64]]:
```

Comment:
I don't know what the `Any` here is for but it always raises a 🚩 to me, so maybe it could be useful to add a comment about why `Any` and not something more specific?